app/src/main/java/com/example/nmc/MainActivity.java

```diff
@@ -13,11 +13,34 @@
 
 public class MainActivity extends AppCompatActivity {
 
+    ListView listView;
+
+
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
```

Comment:
I think you were trying to put this code in ListOfActivity.java, right?

Reply:
Yes. I wasn't sure if it would work if I do that. I thought everything was supposed to be in the mainActivity.

Reply:
Please put the code is ListOfActivity.java

Reply:
Done